css/main.css

```diff
@@ -17,16 +23,22 @@ header {
 	background-image: url('../img/header.png');
 	background-position: bottom;
 	background-repeat: no-repeat;
-
 	height: 360px;
 	max-width: 1200px;
 }
 
-ul {
-	list-style: none;
-	margin: 0;
-	padding: 0;
+.heading-text {
+	color: #ffffff;
 	text-transform: uppercase;
+	font-size: 5em;
```

Comment:
EM is better than PX bit let's use REM
https://learn.javascript.ru/css-units#edinitsa-rem-smes-px-i-em